validator/errorbundler.py

```diff
@@ -280,18 +283,25 @@ def _merge_messages(self, messages, callback, name):
             else:
                 trace.append(message["file"])
 
+            keys = ("column",
+                    "compatibility_type",
+                    "context",
+                    "description",
+                    "editors_only",
+                    "for_appversions",
+                    "line",
+                    "signing_severity",
+                    "tier")
+
+            kw = {key: message[key] for key in keys if key in message}
+
```

Comment:
The main thing is that with all the noise, it wasn't clear to me which keys were being propagated from the old messages, and which were missing/being changed in some other way.

I actually want to get rid of this merging code altogether and just store the filename stack when we push/pop a state, and take care of the filename modifications in `_save_message`, but I didn't want to do anything complicated for something that needed to go out quickly.

And, yes, `editors_only` and `signing_severity` are the two properties that were missing.